docs/setup/settings.asciidoc

```diff
@@ -29,6 +29,27 @@ service that Kibana uses to display map tiles in tilemap visualizations. By defa
 `tilemap.options.attribution:`:: *Default: `"© [Elastic Tile Service](https://www.elastic.co/elastic-tile-service)"`* The map attribution string.
 `tilemap.options.subdomains:`:: An array of subdomains used by the tile service.
 Specify the position of the subdomain the URL with the token `{s}`.
+
+[[regionmap-settings]] `regionmap`:: configuration object to include additional vector layers for use in the <<regionmap,Region Map visualization>>.
```

Comment:
@thomasneirynck in looking at that screenshot, I think we need to update that tilemap.options.attribution link as well! (and any other reference to the tile service)